teuthology/task/selinux.py

```diff
@@ -20,6 +20,15 @@ class SELinux(Task):
     must first be enabled and the filesystem must have been labeled.
 
     On teardown, also checks the audit log for any denials.
+    By default the test will fail for any denials seen in audit.log, for the
```

Comment:
The wording could be wrong, whitelist is still optional like log-whitelist, so that's why i meant by default it may fail unless included in se-whitelist.